projects/fullcalendar/tsconfig.lib.prod.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "./tsconfig.lib.json",
+  "angularCompilerOptions": {
+    "enableIvy": false
```

Comment:
Enabling Ivy is not explicitly required, as it's set to `true` by default ([see official documentation](https://angular.io/guide/angular-compiler-options#enableivy)).

Reply:
By default, this was done by ng-update.